src/main/resources/assets/pvpmode/modules/lotr/configurationComments.properties

```
@@ -2,4 +2,11 @@ enemy_biome_overrides_enabled=If true, the PvP mode enemy biome override conditi
+gear_items_blocked=If true, the gear item blocking system will be enabled, which blocks the usage of certain gear items for players, mostly alignment based. Blocked items can't be used for combat, and they can't be right-clicked. Blocked armor also provides no protection, altough certain features still could work, if the armor is just equipped (to example Galvorn armor would still protect one from arrows). Additional features which prohibit this too can be enabled via the other configuration options. Which items will be blocked can be configured in a separate file. The blocks don't apply for players in creative mode.
+#item_usage_blocked_messages=A message in this list will be randomly chosen and displayed to a player who tries to use a blocked item. If it's empty, the default messages will be used. Duplicated entries and blank lines will be ignored.
+item_usage_blocked_message_cooldown=The cooldown, after which chat messages will be displayed, if blocked items were used. Use -1 to don't display any warnings.
+armor_inventory_check_interval=The interval, in which the armor inventory of a player is checked for blocked armor items. If the armor inventory contains such ones, the player will receive a warning, and eventually, the armor items will be removed (look at the other configuration options). Use -1 to disable the checks.
+blocked_armor_remove_timer=The timer, after which blocked armor items will be removed from the player's armor inventory. The timer will start with the armor inventory check, that did discover these invalid items. Use -1 to disable the removal of the items. What "Remove" exactly means, can be specified with other configuration options. If the armor inventory checks are already disabled, the value of this configuration option is not relevant.
+armor_remove_action=The action that will be done, if blocked armor items are removed from the player's inventory. Regardless of what option is used, the game will first try to store armor items in the player's main or hotbar inventory. If some items can't be stored there, the action specified here will be done. Use 0 to just drop the item on the ground at the player's position, or use 1, to store the item in a hidden inventory. Players then claim these items again from that "hidden" inventory, by typing the command "/pvpvault claim".
```

```suggestion
armor_remove_action=The action that will be used to remove blocked gear items which do not fit in the player's inventory. Use 0 to just drop the item on the ground at the player's position, or use 1, to store the item in a hidden inventory. Players can claim these items again by executing the "/pvpvault claim" command.
```

And I strongy recommend a string (“GROUND”, “VAULT”) or boolean (`drop_removed_armor_to_ground`) approach.

---

See above. I would like an enum, which is for the user like the string you suggested.